src/providers/clojure.rs

```diff
@@ -47,7 +47,9 @@ impl Provider for ClojureProvider {
         let mut build = Phase::build(Some(format!("{build_cmd}; {move_file_cmd}")));
         build.depends_on_phase("setup");
 
-        let start = StartPhase::new("JAR_FILE=$(find ./target -name \"*standalone.jar\"); bash -c \"java $JAVA_OPTS -jar $JAR_FILE\"");
+        let start = StartPhase::new(
```

Comment:
Does the above `move_file_cmd` also need to be updated?

Reply:
It shouldn't be but I'll do another pass today and see if I can make both of them consistent.